core/src/main/java/org/bitcoinj/crypto/DeterministicKey.java

```diff
@@ -149,12 +149,6 @@ public DeterministicKey(List<ChildNumber> childNumberPath,
                 chainCode, HDPath.M(childNumberPath), null, null);
     }
 
-    /** Clones the key */
-    public DeterministicKey(DeterministicKey keyToClone, DeterministicKey newParent) {
```

Comment:
You don't want to `@Deprecate` this constructor?

Reply:
Yes, thanks. It doesn't hurt.